docs/reference-docs/dynamic-fork-task.md

```diff
@@ -3,15 +3,16 @@ sidebar_position: 1
 ---
 
 # Dynamic Fork
+
 ```json
 "type" : "FORK_JOIN_DYNAMIC"
 ```
 
 ## Introduction
 
-A Fork operation in conductor, lets you run a specified list of other tasks or sub workflows in parallel after the fork
+A Fork operation in the conductor, lets you run a specified list of other tasks or sub workflows in parallel after the forked
```

Comment:
Conductor is the name of the tool.  
I think perhaps:
"A Fork operation in Conductor lets you run a specified list of other tasks..."
reads better (capitalized Conductor and removed the comma)